File: 백준/Bronze/4589. Gnome Sequencing/Gnome Sequencing.py
a = int(input())
print("Gnomes:")
for _ in range(a):
  x, y, z = map(int, input().split())

  if x < y and y < z:
    print("Ordered")
  elif x > y and y > z:
    print("Ordered")
  else:
    print("Unordered")

File: 백준/Bronze/4589. Gnome Sequencing/Gnome Sequencing.c
#include <stdio.h>

int main(void) { 
   int a, A, B, C;
   scanf("%d", &a);

   printf("Gnomes:\n");
   for(int i = 0; i < a; i++){
      scanf("%d %d %d", &A, &B, &C);
      
      if(A > B && B > C){
         printf("Ordered\n");
      }
      else if(A < B && B < C){
         printf("Ordered\n");
      }
      else{
         printf("Unordered\n");
      }
   }

   return 0;
}